backend/store/etcd/event_store.go

```diff
@@ -52,8 +50,8 @@ func (s *Store) DeleteEventByEntityCheck(ctx context.Context, entityID, checkID
 	return err
 }
 
-// GetEvents returns the events for an (optional) organization. If org is the
-// empty string, GetEvents returns all events for all orgs.
+// GetEvents returns the events for an (optional) namespaceanization. If namespace is the
```

Comment:
`namespaceanization` 😆 
```suggestion
// GetEvents returns the events for an (optional) namespace. If namespace is the
```